src/machine/usb/adc/midi/midi.go

```diff
@@ -79,7 +96,11 @@ func (m *midi) sendUSBPacket(b []byte) {
 }
 
 // from BulkIn
-func (m *midi) Handler() {
+func (m *midi) TxHandler() {
+	if m.txHandler != nil {
+		m.txHandler()
+	}
```

Comment:
Basically, it should be called when the transmission is completed.
It would be better to treat refactoring of the part that originally existed as a separate PR.

Reply:
The reason that I put the handler first was in case the developer wanted to change the data being sent. But that is perhaps unrealistic and could introduce some latency. I think I will move the handler after the transmit, as has been suggested here.